README.md

```diff
@@ -1,2 +1,56 @@
-# Telegraf-Output-Apache-Arrow-Flight
-External Telegraf plugin for writing time series to Apache Arrow Flight
+# Apache Arrow Flight External Output Plugin
+
+This plugin writes to [ModelarDB-RS](https://github.com/ModelarData/ModelarDB-RS) via the [Apache Arrow Flight protocol](https://arrow.apache.org/docs/format/Flight.html).
+
+## Arrow Schema
+
+The plugin currently supports inserting data points with the following schema:
+
+```rust
+ Field { name: "tid", data_type: Int32, nullable: false, dict_id: 0, dict_is_ordered: false, metadata: None }, 
+ Field { name: "timestamp", data_type: Timestamp(Millisecond, None), nullable: false, dict_id: 0, dict_is_ordered: false, metadata: None }, 
+ Field { name: "value", data_type: Float32, nullable: false, dict_id: 0, dict_is_ordered: false, metadata: None }
+```
+
+## Setting up and running the plugin
+
+To build the binary and run the plugin:
+
+1. Install the latest version of [Go](https://go.dev/doc/install).
+2. Build the binary:
+    * Windows: `go build -o binary/flight.exe cmd/main.go`
+    * Linux/macOS: `go build -o binary/flight cmd/main.go`
+3. Download the latest version of [telegraf](https://github.com/influxdata/telegraf/releases) for your platform. (To see which platform is needed, run the following command: `go env GOOS GOARCH`)
+4. Extract the telegraf executable and configuration to the repository folder.
```

Comment:
I'm not sure I understand, why does a new link change the instructions? 

Should I link to [this](https://docs.influxdata.com/telegraf/v1.21/external_plugins/shim/)? I think that seems like a bit too many instructions to just build and run the plugin.

Reply:
My point was that bullet 4 may not be necessary if we link to https://portal.influxdata.com/downloads/ since platform and format specific installation instructions are provided on that website. However, if you think the instructions are more clear if we keep bullet 4. then that is also Ok for me.